src/tic_tac_toe_pair/rules.clj

```diff
@@ -29,7 +29,7 @@
 
 (defn- is-game-won? [board] (not= nil (get-winning-token board)))
 
-(defn is-game-over? [game]
-  (if (or (is-full? (:board game)) (is-game-won? (:board game)))
+(defn is-game-over? [board]
+  (if (or (is-full? board) (is-game-won? board))
```

Comment:
Could this be reduced to just being the boolean conditional? Do you need to return true/false like that?

Reply:
Yes!!!! Great catch. I'll change it to:

`(defn is-game-over? [board]
  (or (is-full? board) (is-game-won? board)))`

Reply:
That looks perfect

Reply:
Sarah, thank you for your time looking at this. Do you approve this PR? 
I've been guided to make sure to get explicit approval before merging it.

Reply:
@gilroman Oh my gosh, I meant to hit approve but then I got lost in other stuff. Sorry!!!!!!

Reply:
@ssunday No worries at all!!! It was great getting your feedback.